app/models/match.rb

```diff
@@ -392,6 +393,10 @@ def enemy_leaver_char
     char_for_boolean(enemy_leaver)
   end
 
+  def potg_char
```

Comment:
How about spelling it out, since char is already an abbreviation, for clarity? play_of_the_game_char